src/App.js

```diff
@@ -24,6 +25,7 @@ function App() {
           <Route element={<AuthenticateUser />}>
             <Route path='/posts' element={<PostsPage />} />
           </Route>
+        <Route path='/log' element={<DeliveryLogDash />} />
```

Comment:
have you tested if I can go to the DeliveryLogDash ONLY as a logged in user? Seems like I might not need to be authenticated to access this page?